modules/administration/pages/prometheus.adoc

```diff
@@ -0,0 +1,214 @@
+The packages are shipped with {productname} Server and Proxy, but their respective systemd daemons are disabled by default.
+
+These exporter packages are shipped with {productname} Server:
+
+* Node exporter: [systemitem]``golang-github-prometheus-node_exporter``.
+See https://github.com/prometheus/node_exporter.
+* PostgreSQL exporter: [systemitem]``golang-github-wrouesnel-postgres_exporter``.
+See https://github.com/wrouesnel/postgres_exporter.
+* JMX exporter: [systemitem]``prometheus-jmx_exporter``.
+See https://github.com/prometheus/jmx_exporter.
+* Apache exporter: [systemitem]``golang-github-lusitaniae-apache_exporter``.
+See https://github.com/Lusitaniae/apache_exporter).
+
+These exporter packages are shipped with {productname} Proxy:
+
+* Node exporter: [systemitem]``golang-github-prometheus-node_exporter``.
+See https://github.com/prometheus/node_exporter.
+* Squid exporter: [systemitem]``golang-github-boynux-squid_exporter``.
+See https://github.com/boynux/squid-exporter.
+
+
+
+== Prometheus Configuration
+
+Prometheus requires some configuration to collect metrics and set up alarms, or to display metrics graphically in Grafana.
+You can configure Prometheus in the static configuration file at [path]``/etc/prometheus/prometheus.yml``.
+It is important to understand how this file is structured.
+For example:
+
+----
+yaml
+- job_name: 'suse-manager-server'
+  static_configs:
+    - targets:
+      - 'suse-manager.local:9100'  # Node exporter
+      - 'suse-manager.local:9187'  # PostgreSQL exporter
+      - 'suse-manager.local:5556'  # JMX exporter (Tomcat)
+      - 'suse-manager.local:5557'  # JMX exporter (Taskomatic)
+      - 'suse-manager.local:9800'  # Taskomatic
+    - targets:
+      - 'suse-manager.local:80'    # Message queue
+      labels:
+        __metrics_path__: /rhn/metrics
+----
+
+For more information about configuring Prometheus, see the official Prometheus documentation: https://prometheus.io/docs/prometheus/latest/configuration/configuration/
+
+
+
+== Monitoring Managed Systems
+
+Prometheus metrics exporters can also be used on managed client systems.
+The packages are available from the {productname} client tools channels, and can be enabled and configured directly on the {productname} {webui}.
+Currently, two exporters are supported:
+
+* Node exporter: [systemitem]``golang-github-prometheus-node_exporter``.
+See https://github.com/prometheus/node_exporter.
+* PostgreSQL exporter: [systemitem]``golang-github-wrouesnel-postgres_exporter``.
+See https://github.com/wrouesnel/postgres_exporter.
+
+Installing and configuring exporters is done using a Salt formula.
+
+
+
+== Enable and Configure Monitoring
+
+
+.Procedure: Enabling Monitoring
+
+. In the {productname} {webui}, navigate to menu:Admin[Manager Configuration > Monitoring].
+. Click btn:[Enable services].
+
+image::monitoring_enable_services.png[scaledwidth=80%]
+
+
+.Procedure: Configure Prometheus
```

Comment:
This should be called "Configure Monitoring formulas", as we are not configuring Prometheus itself, just enabling monitoring formulas for a given system. Enabling the formula is a required step to configure the exporters (below).